tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -495,6 +495,19 @@ module.exports = {
       }
       return this.saveChanges()
     },
+    /**
+     * @param {string} days number of days to be added or subtracted from current date
+     *
+     * @return {Promise<*>}
+     */
+    attemptToChangeCollaboratorExpiryDateToDisabledValue: async function (days) {
```

Comment:
why not using `setExpirationDate()` from the datepicker? that will return `false` if the date is disabled

Reply:
fixed